allennlp/models/vision_text_model.py

```diff
@@ -0,0 +1,256 @@
+import logging
+from copy import deepcopy
+from typing import Dict, List, Optional
+
+from overrides import overrides
+import numpy as np
+import torch
+
+from allennlp.data import TextFieldTensors, Vocabulary
```

Comment:
It's safer to import these from the modules they are defined in. I haven't quite figured out yet why this happens, but we often have problems with circular imports otherwise.

Reply:
Updated